### ISSUE TITLE
Analytics MATOMO

### ISSUE THREAD

**Issue by andriacap** [PnX-SI/GeoNature-atlas#499]
Bonjour,

Je relance le sujet concernant une alternative à l'outil de mesure de statistiques web (MATOMO anciennement PIWIK). Pour rappel MATOMO est une solution libre et opensource et reconnu officiellement par la CNIL .
Est ce qu'un développement pour laisser le choix à l'utilisateur de son outil de mesure statistiques serait accepté ?

On pourrait alors avoir un paramètre de config supplémentaire en renseignant les informations liés à MATOMO ?

Merci d'avance pour vos retours

**Comment by camillemonchicourt** [comment]
Oui, c'est assez gênant de fournir des outils qui ne fonctionnent qu'avec Google Analytics, erreur de jeunesse...

On a travaillé sur le sujet récemment pour Geotrek-rando-v3, et au lieu d'ajouter des paramètres, on a mis en place un système où chacun peut injecter les scripts qu'il souhaite : https://github.com/GeotrekCE/Geotrek-rando-v3/issues/459

Plus complexe à mettre en place, mais plus souple et évolutif.

Dans notre cas, on peut se permettre d'ajouter un paramètre pour l'ID Matomo, mais à priori il y a plusieurs manières et plusieurs niveaux de détail dans la manière d'intégrer Matomo, donc à voir si c'est possible et pertinent.

**Comment by camillemonchicourt** [comment]
On partirait plutôt sur un script que chacun peut renseigner avec l'outil de suivi qu'il veut (en fournissant des exemples), et donc supprimer le paramètre et le code spécifique à Google Analytics, ni ajouter un paramètre et code spécifique à Matomo.
A bien documenter donc.

**Comment by camillemonchicourt** [comment]
C'était déjà possible de le faire en intégrant les scripts MATOMO dans le template custo du footer.
Cela a été précisé, clarifié et complété dans la 1.7.0.
En lien avec le travail sur le RGPD et le consentement des cookies (non nécessaire si on utilise Matomo) : https://github.com/PnX-SI/GeoNature-atlas/blob/master/docs/cookies_rgpd.rst
